src/Storage/Listener/LocalListener.php

```diff
@@ -29,7 +27,7 @@ class LocalListener extends AbstractListener {
  * @var array
  */
 	protected $_defaultConfig = [
-		'pathBuilder' => 'BasePath',
+		'pathBuilder' => 'Local',
```

Comment:
There's no need to do that right now, because Local path builder seems to have the same config as a Base. But we could prepare the code for extending default config for the future usign this trait. I can take care of this later, now I'm trying to move file storage code to the behavior :)